addons/mail/static/src/models/attachment_viewer_viewable.js

```diff
@@ -0,0 +1,176 @@
+/** @odoo-module **/
+
+import { registerModel } from "@mail/model/model_core";
+import { attr, one } from "@mail/model/model_field";
+
+registerModel({
+    name: "AttachmentViewerViewable",
+    identifyingFields: [["attachmentOwner"]],
+    recordMethods: {
+        download() {
+            if (this.attachmentOwner) {
```

Comment:
Aren't all the conditions on `attachmentOwner` unnecessary in this file?
The override is checking for its own type and then just calling super if needed, which would always fall back to having an attachment?

Reply:
True